docs/usage/configuration-options.md

```diff
@@ -3385,6 +3385,22 @@ In case there is a need to configure them manually, it can be done using this `r
 
 The field supports multiple URLs but it is datasource-dependent on whether only the first is used or multiple.
 
+## renovateCompatibility
+
+This capability exists so that Renovate maintainers can change default behavior without forcing all existing users to adopt the new behavior immediately.
+Although Renovate adheres to SemVer and limits "breaking" behavior changes to major releases, the majority of _users_ of Renovate are end users in repositories and they do not have control over which major release of Renovate is running or when that major upgrade is performed.
```

Comment:
```suggestion
Renovate follows SemVer and limits "breaking" behavior changes to major releases.
This works well for users who self-host Renovate, because they control when Renovate is updated.
But most _users_ of Renovate are end users in repositories, they can't control which major release of Renovate is running or when Renovate updates.
```